index.html

```diff
@@ -8,8 +8,39 @@
 <div class="block">
   <h1><span class="header-inside">Projects</span></h1>
   <div id="projects-panel">
+    <table class="projects">
+      <noscript>
+        {% for p in site.data.projects %}
+        {% assign project = p[1] %}
+        <tbody>
+          <tr>
+            <td colspan="2" class="title">
+              <span class="proj"><a href="{{project.site}}">{{project.name}}</a></span>
+            </td>
+          </tr>
+          <tr>
+            <td class="details">
+              <p class="label"><a href="{{ project.upforgrabs.link }}" title="View open issues for {{ project.name }}">{{ project.upforgrabs.name }}</a></p>
```

Comment:
You mention open issues - however, it's guaranteed that that link will take you to a list of issues for that project. Some have added links to their contributors' guidelines, and some to a discussion board (iirc).

It might be better to generalize this verbiage, to something like "Contribute to XYZ project!"